packages/@okta/vuepress-site/docs/concepts/feature-lifecycle-management/index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Feature Lifecycle Management
+---
+
+# Feature Lifecycle Management
+
+This page will cover a few key concepts that underpin the [Okta Features API](/docs/reference/api/features/). The Features API allows you to manage self-service Features, as well as their Feature dependents and dependencies. Special restrictions apply to the behavior of the API with respect to Beta features.
+
+## Self-service Features
+
+Self-service Features are Features that can be enabled or disabled by an Okta admin directly. Features that are not self-service can only be enabled or disabled by getting in touch with Okta Support. The [Okta Features API](/docs/reference/api/features/) is used to list, enable, and disable self-service Features.
+
+> **Note**: A Beta Fature that is marked as self-service will stay as self-service even if it's closed and temporarily unavailable to be enabled. For example, if an [open Beta](#beta-features) transitions into a closed Beta, the Feature will remain self-service. This means that it will appear in the list of available self-service Features, but you will not be able to enable it without contacting Support.
+
+Only Beta and Early Access (EA) Features are considered self-service. Once a Feature transitions from either of these states to Generally Available (GA), it will no longer appear as an available self-service Feature.
+
+## Beta Features
+
+Though all available Betas qualify as "self-service Features", Betas can be either open or closed. Open Betas can be both enabled or disabled using the API, however closed Betas can only be disabled. If a Feature has Feature dependencies that are in Closed Beta, then you will not be able to enable it. If you'd like to enable a closed Beta, you should [contact Support](mailto:support@okta.com). Enabling an Open Beta will trigger an email to the admin who performed the action.
```

Comment:
do we need to explain what Open and Closed Betas are? I.e. do we need to say Beta Flags will be closed to for self-service once we've received enough participation - or is that revealing too much?

Reply:
i think that'd be revealing tmi